tests/copilot_helper/test_command_copilot.py

```diff
@@ -67,66 +67,76 @@
 
 
 class TestMakeAddonCommand:
+    @patch(
+        "dbt_copilot_helper.utils.versioning.running_as_installed_package",
+        new=Mock(return_value=True),
+    )
+    def test_validate_version_called_once(self, validate_version):
```

Comment:
It felt like it was worth separating this out to it's own test and not worrying about it in the rest of them.